src/main/webapp/styles/recommendation-map.css

```diff
@@ -61,21 +61,25 @@ h2, h3, p, label {
   text-align: center;
 }
```

Comment:
nit: I think we don't need this class anymore?

Reply:
Good spot, thanks!